tasks/search.yml

```diff
@@ -5,4 +5,6 @@
   shell: "php symfony search:populate 1>/dev/null"
   args:
     chdir: "{{ atom_path }}"
-  ignore_errors: "yes" # segfault in php7, not sure yet why!
+  environment:
```

Comment:
does it completely build the index even though it segfaults? Maybe need to clarify in the comment